src/actions/kloudio/kloudio.ts

```diff
@@ -4,7 +4,7 @@ import * as uuid from "uuid"
 import * as winston from "winston"
 import * as Hub from "../../hub"
 
-const signedUrl: any = process.env.KLOUDIO_SIGNED_URL
+const signedUrl = "https://api.kloud.io/v1/tools/signed-url-put-object?key="
```

Comment:
why was this reverted back?

Reply:
Initially I had thought both the variables had to be changed but since the previous PR comments & Shohei's email only mentioned "API_URL" I reverted it back since it was not asked.

Reply:
Hey Anirrudh, my apologies. I think @phillipperalez was asking for both - basically not hard-coding URLs in general for these two variables.

Reply:
Hi Shohei! Thanks for the clarification. I will make the changes and update the PR.

Reply:
@snarron @phillipperalez I have made the changes by reading both values from env variables. The test file is using a test url not the live url.